tests/docker_ssi/test_docker_ssi_crash.py

```diff
@@ -42,7 +41,6 @@ def setup_crash(self):
     )
     @irrelevant(context.library == "python" and context.installed_language_runtime < "3.7.0")
     @irrelevant(context.library == "nodejs" and context.installed_language_runtime < "17.0")
-    @bug(context.library >= "python@3.0.0.dev", reason="INPLAT-603")
```

Comment:
Is the bug exists in released version ? if yes, could you skip the test for those release ? otherwise, backports will be painful